src/uu/dircolors/README.md

```diff
@@ -9,10 +9,16 @@ dircolors -b > /PATH_TO_COREUTILS/tests/fixtures/dircolors/bash_def.expected
 dircolors -c > /PATH_TO_COREUTILS/tests/fixtures/dircolors/csh_def.expected
 ```
 
+Apply the patches to include more terminals that support colors:
+
+```shell
+git apply /PATH_TO_COREUTILS/src/uu/dircolors/alacritty-supports-colors.patch
+```
+
```

Comment:
Why is this step necessary? The change is already applied by this commit, so this instruction is meaningless.

Suggestion: Remove this instruction, and remove the patchfile.

Reply:
It is necessary because this README explains how to update the fixtures from the internal database, so the next person who does it should also do this step. Otherwise, the changes will be lost and the tests will fail again next time.

Reply:
Why would changes in git be lost?

Reply:
If one updates the test fixtures, why wouldn't they commit the changes? They will, obviously, and this patch would be lost. Therefore, we need to tell them to re-apply it.
It seems quite a simple matter to me. Maybe I'm not expressing myself well, if so, please tell me.

Reply:
Is it possible that you didn't notice that the change to the fixture is already in the commit? https://github.com/uutils/coreutils/pull/6725/files#diff-26d0675eb8206b8804aec078728efbfb465417b19b82bf606811a217f9580bc3

Reply:
I know, I made the commit. :sweat_smile: 

Okay. Let me try to explain this with an example.

1. Our fellow developer, let's call them Alice, notices something odd in uutils `dircolors`.
2. They make some changes to fix the behavior.
3. At this point, they want to run the tests to make sure they didn't break anything.
4. Alice is a fan of `dircolors` and knows that a new version of the GNU counterpart has just been released.
5. They recall seeing the [README of uutils `dircolors`](https://github.com/uutils/coreutils/blob/main/src/uu/dircolors/README.md), titled "How to update the internal database".
6. Since a new release of GNU `dircolors` is out, they decide to [follow the steps to update the test fixtures](https://github.com/uutils/coreutils/blob/317f6d9a4abd63bc826e952bd80dd171e1d98548/src/uu/dircolors/README.md?plain=1#L6-L9).
7. Now, they run the tests and... A test is failing!
8. However, it is not Alice's fault; _their code did not actually break anything_.

So why did a test fail? The problem is that, at step 6, Alice updated the test fixtures overwriting the patch that adds Alacritty to the list of terminals that support colors.

Conclusion: We should tell Alice to apply the patch every time they regenerate the test fixtures. Also, we kindly provide them the patch file.

Reply:
I don't think it is needed.
when we will refresh src/uucore/src/lib/features/colors.rs
we will see the diff and keep the alacritty and other changes:)